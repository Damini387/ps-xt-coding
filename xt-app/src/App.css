.App {
  width: 100%;
  max-width: 1440px;
  margin: 10px auto;
}

.doneBy {
  display: flex;
  flex-direction: column;
  justify-content: center;
  align-items: center;
}

.center {
  display: flex;
  justify-content: center;
  margin: 5px 0;
}

@media screen and (min-width: 700px) {
  .doneBy {
    flex-direction: row;
  }

  .devName {
    margin-left: 5px;   
   }
}