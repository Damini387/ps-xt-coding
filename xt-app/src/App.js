import React from 'react';
import './App.css';
import LaunchProgram from './components/launchProgram/index';

function App() {
  return (
    <div className="App">
      <LaunchProgram />
      <h3 className="center">
        Developed By: <span>Damini Upadhyay</span>
      </h3>
    </div>
  );
}

export default App;
