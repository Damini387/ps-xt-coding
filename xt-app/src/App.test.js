import React from 'react';
import { shallow } from 'enzyme';
import App from './App';

let wrapper = shallow(
	<App>
		<div className="doneBy">
			<h3 className="center"> Developed By: </h3>
			<span className="devName"> Damini Upadhyay </span>
		</div>
	</App>);

describe('App', () => {

	it('should render the App Component correctly', () => {
		expect(wrapper).toMatchSnapshot();
	});

	it('renders the App children', () => {
		expect(wrapper.find('.devName').text()).toContain("Damini Upadhyay");
	});

});