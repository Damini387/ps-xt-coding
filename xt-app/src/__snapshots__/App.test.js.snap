// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render the App Component correctly 1`] = `
<div
  className="App"
>
  <LaunchProgram />
  <div
    className="doneBy"
  >
    <h3
      className="center"
    >
       Developed By: 
    </h3>
    <span
      className="devName"
    >
       Damini Upadhyay 
    </span>
  </div>
</div>
`;
